receiver/prometheusreceiver/README.md

```diff
@@ -68,6 +68,11 @@ prometheus --config.file=prom.yaml
 ```shell
 "--feature-gates=receiver.prometheusreceiver.UseCreatedMetric"
 ```
+- `receiver.prometheusreceiver.EnableCreatedTimestampZeroIngestion`: Enables the Prometheus feature flag [created-timestamps-zero-injection](https://prometheus.io/docs/prometheus/latest/feature_flags/#created-timestamps-zero-injection). Currently, this behaviour is enabled by default. To disable it, use the following feature gate option:
```

Comment:
Since prometheus doesn't do this by default, we shouldn't either: https://github.com/prometheus/prometheus/blob/24057883a14c597d70f46612e4b8fb5c9ad58d62/cmd/prometheus/main.go#L265-L266